### ISSUE TITLE
Typo in contrib/celery.py

### ISSUE THREAD

**Issue by fidelitas** [viewflow/viewflow#168]
```
python3.5/site-packages/viewflow/contrib/celery.py:31: in async
    apply_kwargs['eta'] = self.flow_Task._eta(self.task)
E   AttributeError: 'JobActivation' object has no attribute 'flow_Task'
```

**Comment by kmmbvnr** [comment]
Already fixed in latest PRO

**Comment by fidelitas** [comment]
@kmmbvnr OK, found the issue. We needed to pip install with --no-cache-dir.

**Comment by codingjoe** [comment]
@kmmbvnr thanks! That was strange tho. Was the wheel edited after the release? Had to dump the cache pip cache on all our systems.
Anyways, working now 👍

**Comment by kmmbvnr** [comment]
This was fixed on 0.10.8, i think just upgrading up to 0.10.10 was fixed the problem
